galsim/base.py

```diff
@@ -269,18 +273,9 @@ def __truediv__(self, other):
     # Make a copy of an object
     def copy(self):
         """Returns a copy of an object.
```

Comment:
Correct me if I'm wrong, but I think that if the GSObject has an RNG then the RNG is being copied, not duplicated, so we can't expect identical images (e.g., for a RealGalaxy).  This seems like a caveat that belongs in the docstring.